fire/README.md

```diff
@@ -55,6 +61,13 @@ To open the reported post on it's site:
 5. Fire sees the hash, saves your token, then closes the new window.
 6. You go back to the old window, it detects a token, and shows you a "success" message.
 
+# Common Failures
+
+1. When you first install FIRE, you probably have to reload the chat room before you see the clickable 🔥 FIRE decorations
+2. Forgetting to authorize FIRE. See the previous section.
+3. If you are a moderator on a site, it will refuse to flag on your behalf as a security precaution.
```

Comment:
This sounds as though FIRE will not work _at all_ for you if you're a moderator. In reality, it will only fail on sites that you moderate. Maybe something like:

> FIRE will not submit flags on any site where you are a diamond moderator (♦) as a security precaution.

Reply:
I'm about to fix this and another thing